### PR TITLE
Added timezone in examples

### PR DESCRIPTION
**Pull request by alessandrocavalli**
Related to [CAM-14519](https://jira.camunda.com/browse/CAM-14519)

### CODE REVIEW

**Comment by alessandrocavalli** [comment]
CAM-14519

File: content/reference/bpmn20/events/timer-events.md [hunk 1]
```diff
@@ -145,23 +145,23 @@ Note: A subprocess cannot have a timer start event.
...
 <startEvent id="theStart">
   <timerEventDefinition>
-    <timeCycle>R4/2016-03-11T12:13/PT5M</timeCycle>
+    <timeCycle>R4/2022-03-11T12:13+01/PT5M</timeCycle>
   </timerEventDefinition>
 </startEvent>
 ```
 
-and this process will start once, on a selected date:
+and this process will start once, on a selected date, at Greewich time:
```
Comment (garima-camunda) [inline content/reference/bpmn20/events/timer-events.md:160]: @alessandrocavalli - There is a typo in "Greewich". Can you correct that?

File: content/reference/bpmn20/events/timer-events.md [hunk 2]
```diff
@@ -145,23 +145,23 @@ Note: A subprocess cannot have a timer start event.
...
 <startEvent id="theStart">
   <timerEventDefinition>
-    <timeCycle>R4/2016-03-11T12:13/PT5M</timeCycle>
+    <timeCycle>R4/2022-03-11T12:13+01/PT5M</timeCycle>
   </timerEventDefinition>
 </startEvent>
 ```
 
-and this process will start once, on a selected date:
+and this process will start once, on a selected date, at Greenwich time:
```
Comment (mboskamp) [inline content/reference/bpmn20/events/timer-events.md:159]: We used the term UTC before. Why not stick with it here? :)
```suggestion
And this process will start once, on a selected date, at UTC time:
```

File: content/reference/bpmn20/events/timer-events.md [hunk 3]
```diff
@@ -145,23 +145,23 @@ Note: A subprocess cannot have a timer start event.
 
 Note: A timer start event is scheduled as soon as process is deployed. There is no need to call `startProcessInstanceBy...`, although calling start process methods is not restricted and will cause one more start of the process at the time of the `startProcessInstanceBy...` invocation.
 
-The XML representation of a timer start event is the normal start event declaration, with a timer definition sub-element. The following example process will start 4 times, in 5 minute intervals, starting on 11th of March 2016, at 12:13 (24 hour clock system):
+The XML representation of a timer start event is the normal start event declaration, with a timer definition sub-element. The following example process will start 4 times, in 5 minute intervals, starting on 11th of March 2022, at 12:13 (UTC +01, 24 hour clock system):
```
Comment (mboskamp) [inline content/reference/bpmn20/events/timer-events.md:148]:
```suggestion
The XML representation of a timer start event is the normal start event declaration, with a timer definition sub-element. The following example process will start 4 times, in 5 minute intervals, starting on 11th of March 2022, at 12:13 (UTC+1, 24 hour clock system):
```

**Review by mboskamp — COMMENTED** [review]
Looks good overall. Nice improvement. I only added two minor suggestions. Let me know what you think ;)

**Comment by mboskamp** [comment]
Hey @alessandrocavalli, in the CAM ticket we agreed to include the improvements I suggested in this PR. Now you merged the PR without those improvements. Is there a specific reason why?

**Comment by mboskamp** [comment]
As discussed with @alessandrocavalli, we will leave the commit as-is and close the ticket.

**Pull request merged**